lib/src/ovis_util/util.h

```diff
@@ -116,6 +116,11 @@ struct attr_value_list {
 
 /**
  * \brief Get the value of attribute \c name
+ * Values will be fully substituted by str_repl_env_vars before
+ * being returned.
+ * \param name the key to search for in av_list.
+ * \return the first value with key name, or NULL if there is no
```

Comment:
According to the code, `av_value()` returns an empty string if the key doesn't exist or it cannot expand the value.

Reply:
util.c lines 450 and 465 disagree with you. can you explain?